tests/virtualization/xen/patch_guests.pm

```diff
@@ -17,24 +17,26 @@ use testapi;
 use qam 'ssh_add_test_repositories';
 use utils;
 use xen;
+use version_utils;
+use virt_autotest::kernel;
 
 sub run {
     my ($self) = @_;
-    my $version = get_var('VERSION');
+    my ($host_running_version, $host_running_sp) = get_sles_release();
```

Comment:
A big :+1: for the refactoring using the new `get_sles_release` routine. Like that!